.github/workflows/tfplugindocs.yaml

```diff
@@ -33,9 +35,26 @@ jobs:
               with:
                   extra_args: --hook-stage=manual --all-files
 
+
+            - name: Temporarily disable "include administrators" branch protection
+              uses: benjefferies/branch-protection-bot@1.0.9
+              if: always()
+              with:
+                  access_token: ${{ secrets.REPO_PAT }}
+                  enforce_admins: false
+
             - name: Commit generated docs
-              uses: stefanzweifel/git-auto-commit-action@v4
+              uses: stefanzweifel/git-auto-commit-action@v5
               with:
-                  # using ci: prefix to hide this commit from changelog
+                  # [skip ci] is NOT added to ensure that new workflow runs (release-please needs to update its PR)
+                  # This should not add an infinite loop because the next time docs are generated they will be the same
+                  # https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs
```

Comment:
Just to reiterate: once the commit is pushed, it will re-trigger all the workflows.
I did it on purpose and documented it :)
Let me know if you're against it or have better idea.

Reply:
Cannot we skip whole pipeline, if last PR was created by automated commit?
This infinite loops is a bit scary

Reply:
@arxeiss we can, but is this something you really want?
Because in this case you need not to forget to update PR created by release-please.
If you have a workaround for this manual action - I will add `[no ci]` to commit message, and will not trigger any GitHub workflows.

Reply:
OK, keep it as is, if it works